addons/binding/org.openhab.binding.avmfritz/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,37 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: avmFritz Binding
+Bundle-SymbolicName: org.openhab.binding.avmfritz;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/jackson-core-2.3.2.jar,
+ lib/jackson-databind-2.3.2.jar,
+ lib/jackson-module-jaxb-annotations-2.3.2.jar
+Import-Package: com.google.common.collect,
+ org.apache.commons.lang,
+ org.apache.commons.lang.builder,
+ org.eclipse.jetty.client,
+ org.eclipse.jetty.client.api,
+ org.eclipse.jetty.client.util,
+ org.eclipse.jetty.http,
+ org.eclipse.jetty.io,
+ org.eclipse.jetty.util,
+ org.eclipse.jetty.util.component,
+ org.eclipse.jetty.util.ssl,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.events,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.types,
+ org.jupnp.model.meta,
+ org.jupnp.model.types,
+ org.openhab.io.net.http,
```

Comment:
Just noticed that you use this from the compatibility bundle - please use the ESH version instead, i.e. ```org.eclipse.smarthome.io.net.http```.